packages/remix-deno/server.ts

```diff
@@ -47,7 +46,7 @@ export async function serveStaticFiles(
     cacheControl?: string | ((url: URL) => string);
     publicDir?: string;
     assetsPublicPath?: string;
-  }
+  },
 ) {
   const url = new URL(request.url);
 
```

Comment:
Let's throw a `FileNotFoundError` instead of whatever `Deno.readFile` throws when it can't find something.